### ISSUE TITLE
How to set the app icon on MacOS 10.13.4.

### ISSUE THREAD

**Issue by AllenDang** [sciter-sdk/go-sciter#150]
I'm on MacOS 10.13.4. I followed the [instruction](https://github.com/therecipe/qt/wiki/Setting-the-Application-Icon#setting-the-icon-on-macos) of a qt binding.

But it doesn't work anymore.

It seems MacOS changes the code sign mechanism, I have to create code signature of entire .app folder in order to show the app icon.

My app is very close to release, the last thing is to set a proper icon. Any thoughts about how I can do that?

**Comment by pravic** [comment]
The [Inspector](https://github.com/c-smile/sciter-sdk/tree/master/bin.osx/inspector.app/Contents/Resources) app from Sciter SDK looks as if it contains icon.

**Comment by AllenDang** [comment]
No, I tried that. Using a existing app which doesn't have code signature, replace the executable file, the icon, and modify the info.plist. Unfortunately this doesn't work out.

**Comment by pravic** [comment]
Does Inspector.app have a valid icon under your MacOS? If yes, ask Andrew @c-smile how Inspector was signed.

**Comment by AllenDang** [comment]
Yes, Inspector.app has a valid icon. But I think it is compiled by xcode and it will sign automatically.

**Comment by AllenDang** [comment]
So.... any thoughts? It sad that we cannot find a way to set proper application icon. This is a very good (the best in my opinion) cross platform GUI framework in entire go ecosystem. 
But this small issue blocks the opportunity to create commercial product. Really hope this can be solved.
